packages/tasks/src/model-libraries.ts

```diff
@@ -200,6 +200,13 @@ export const MODEL_LIBRARIES_UI_ELEMENTS = {
 		filter: true,
 		countDownloads: `path:"pytorch_model.bin"`,
 	},
+	SenseVoice: {
+		prettyLabel: "SenseVoice",
+		repoName: "SenseVoice",
+		repoUrl: "https://github.com/FunAudioLLM/SenseVoice",
+		filter: false,
+		countDownloads: `path:"model.pt"`,
```

Comment:
No need to add this rule. The repo has a `config.yaml` file which is part of the [default set](https://huggingface.co/docs/hub/models-download-stats#which-are-the-query-files-for-different-libraries)